pkg/kurl/join_cert.go

```diff
@@ -127,6 +142,18 @@ func parseCertGenOutput(logs []byte) (string, error) {
 	return "", fmt.Errorf("key not found in %d bytes of output", len(logs))
 }
 
+func getConfigMap(namespace string) *corev1.ConfigMap {
```

Comment:
nit pick: since this is part of the general `kurl` package, name conflict might happen in the future. would be nice to name it something like `joinCertGenConfigMap` or something

Reply:
agreed